ampd/src/grpc/status.rs

```diff
@@ -270,4 +287,28 @@ mod tests {
             Code::Internal
         );
     }
+
+    #[test]
+    fn tofnd_errors_to_status() {
+        assert_eq!(
+            tofnd::Error::GrpcRequest(tonic::Status::permission_denied("permission denied"))
+                .into_status()
+                .code(),
+            Code::PermissionDenied
+        );
+        assert_eq!(
+            tofnd::Error::InvalidKeygenResponse.into_status().code(),
+            Code::Internal
+        );
+        assert_eq!(
+            tofnd::Error::InvalidSignResponse.into_status().code(),
+            Code::Internal
+        );
+        assert_eq!(
+            tofnd::Error::ExecutionFailed("key generation failed".to_string())
+                .into_status()
+                .code(),
+            Code::Internal
+        );
+    }
```

Comment:
seems like this is just a slightly different format than the action `from` implementation. Use goldie instead?

Reply:
(seems like existing tests followed this format already, so would be good to change those as well)